File: Beecrowd/5 - Mathematics/1161 - Factorial Sum/Python/1161_FactorialSum.py
import sys
import math


input = sys.stdin.read()
lines = input.strip().split("\n")

for line in lines:
    a, b = map(int, line.split())
    sum = 0

    for i in range(2):
        n = a if i == 0 else b
        fact = math.factorial(n)
        sum += fact

    print(sum)

File: Beecrowd/5 - Mathematics/1161 - Factorial Sum/1161_FactorialSum.c
#include <stdio.h>

int main() {
    long long a, b;
    
    while (scanf("%lld %lld", &a, &b) != EOF) {
        long long sum = 0;

        for (int i = 0; i < 2; i++) {
            long long n = (i == 0) ? a : b;
            long long fact = 1;

            for (int j = 1; j <= n; j++) {
                fact *= j;
            }

            sum += fact;
        }

        printf("%lld\n", sum);
    }

    return 0;
}
